include/fastgltf/types.hpp

```diff
@@ -1309,8 +1309,29 @@ namespace fastgltf {
          */
         std::variant<TRS, TransformMatrix> transform;
 
+        /**
+         * Only ever non-empty when EXT_mesh_gpu_instancing is enabled and used by the asset.
+         */
+        pmr::SmallVector<std::pair<std::pmr::string, std::size_t>, 4> instancingAttributes;
```

Comment:
I want extensions to be more or less invisible if the user is not using any of them, so having 4 counts of the std::pair *always* part of the Node struct seems weird and like a waste of memory to me when people are not using the extension. This is actually 128 bytes for all the pairs. (24 bytes for string, 8 bytes for size_t, each times four).

I'd want either one of these:
- Wrap the SmallVector in a `unique_ptr` so that this is only 8 bytes. I do this with all other extensions, too.
- Simply use a ordinary `pmr::vector` instead of the `SmallVector`. This is still 24 bytes, but would drastically reduce it from 152 bytes.

Reply:
I've replaced it with a pmr::vector.